packages/chat-client/src/controllers/engine.ts

```diff
@@ -831,6 +838,7 @@ export class ChatEngine extends IChatEngine {
       });
 
       const authorAccount = cacao.p.iss.split(":").slice(2).join(":");
+      const receipientAccount = cacao.p.aud.split(":").slice(2).join(":");
```

Comment:
Can we consider using cacao utils for extracting those values ?
If the spec changes it'll be a PITA if we have to update those across packages.
But if we move those formatting function into cacao utils, we'll just have to update once.

Reply:
The utils in `@walletconnect/cacao` do not cover `.split(":").slice(2).join(":"), but they should, I'll look into opening a PR for that